crates/task-impls/src/builder.rs

```diff
@@ -78,7 +78,7 @@ impl<TYPES: NodeType, Ver: StaticVersionType> BuilderClient<TYPES, Ver> {
     /// If the URL is malformed.
     pub fn new(base_url: impl Into<Url>) -> Self {
         Self {
-            inner: Client::builder(base_url.into().join("block_info").unwrap())
+            inner: Client::builder(base_url.into().join("bundle_info").unwrap())
```

Comment:
This will make Client incompatible with legacy builders. Real marketplace builder also mounts its API on `block_info` actually, so that too.
Is it possible to mount two APIs on the same path in SimpleBuilderSource?

Reply:
Yeah, this is the part I need to figure out how to do properly -- we can't put two on the same endpoint in a single builder.

But if the real builder is serving at `block_info`, maybe it would be better to just have multiple simple builders -- one for each version? I'll look at this again and continue on Zulip

Reply:
Updated: we now have multiple clients at different module paths, which are defined as constants in `types/src/constants.rs`